### ISSUE TITLE
Logs and Shell are not available in the new version

### ISSUE THREAD

**Issue by seilorjunior** [kontena/lens#6983]
On the latest version we have access to go on the logs or shell in the pod.

With this new version this kind of access was removed. why?

BR
Seilor

**Comment by mateusz-kolecki** [comment]
Same for me. After cloning the repo and building `v6.3.0` the buttons are gone. Building and installing `v6.2.5` solves the problem.

![obraz](https://user-images.githubusercontent.com/4613412/213451399-e58570f1-75e5-4a7f-a732-57b7ec1893bd.png)

**Comment by mateusz-kolecki** [comment]
Ok, it is not a bug. It is a "feature". The buttons were removed intentionally by the authors and moved as a part of a non-MIT licensed solution. More info here: https://github.com/MuhammedKalkan/OpenLens

**Comment by simonberger** [comment]
> Ok, it is not a bug. It is a "feature". The buttons were removed intentionally by the authors and moved as a part of a non-MIT licensed solution. More info here: https://github.com/MuhammedKalkan/OpenLens

Not sure if this is really the reason here. We have two different clusters. The aks one shows those buttons, the other one don't. See also https://github.com/MuhammedKalkan/OpenLens/issues/81#issuecomment-1380191982
The issue of Openlens 6.3.0 was also a completely removed pod entry and no specific pod actions

I cannot assess how the behavior should be and where something is implemented, but the differences between the cluster types is at least suspicious.

**Comment by seilorjunior** [comment]
@mateusz-kolecki i am using this free version from https://github.com/MuhammedKalkan/OpenLens and also it does not have the buttons.

**Comment by guenhter** [comment]
I'm not sure if this was really intentional. When I look at https://github.com/MuhammedKalkan/OpenLens/releases nothing stats that the logs are removed... @mateusz-kolecki Do you know more here what the purpose was, or what merge request removed that?

**Comment by guenhter** [comment]
Seems to be a duplicate of this https://github.com/lensapp/lens/issues/6823

**Comment by simonberger** [comment]
I don't know in which circumstances the Pods entry was available in 6.3.0 and just the actions missing (in OpenLens). I never had any Pods entry in 6.3.0. This recovered in 6.4.0 Alpha, but the Shell + Log Actions were then unavailable in one cluster type.
I am on Mac Intel.

**Comment by MalHaak** [comment]
@alebcay/openlens-node-pod-menu

Add it to your extensions. All they did was remove links to external repo's from their main repo. Ya'll are making a storm in a teacup.

**Comment by jamiecarl** [comment]
@MalHaak Not really.  For those of us that just like to use software and don't have time to fart-ass about reading Github issue comments to finally figure out that it's been removed "as a default" but is still availble as an extension, and then when we go read the list of available extensions linked from the app and find that the Node Pod Menu extension isn't even listed and so we can't figure out how to fix it for ourselves without going through pages and pages of Github issues, well, it's a bit of a fucking issue, mate.  I've got better things to do.

I've created  a PR to update the extension list to include Node Pod Menu (see here: https://github.com/lensapp/lens-extensions/pull/21), which is the absolute minimum that should be done to make this situation simpler for users, but it's been a couple of days and no one has even looked at it, so I'm not confident anything is going to be done with it.  This whole situation is an absolute clusterfuck, in my opinion.

**Comment by MalHaak** [comment]
@JAZ-013 One google search. That's what it took for me to go from "Oh why is this broken" to "Back to work". Could they have done a better job of warning people, sure. But it's not really that big of a deal and their policy does make sense. 

But I see you've already fetched the pitchforks and lit your torches.

**Comment by jamiecarl** [comment]
> @JAZ-013 One google search. That's what it took for me to go from "Oh why is this broken" to "Back to work". Could they have done a better job of warning people, sure. But it's not really that big of a deal and their policy does make sense.
> 
> But I see you've already fetched the pitchforks and lit your torches.

Well, now that everyone here has been discussing it and have done the hard work for you, I'm sure it was incredibly easy to figure it out.  But sure, go ahead and have a go at someone that is trying to help.  Great work, hero.

**Comment by akundrock** [comment]
Thank you for opening this issue and posting a solution!
